components/MainNavigation/index.js

```diff
@@ -38,7 +35,7 @@ const MainNavigation = () => {
           <Stack.Screen
             name="Root"
             component={BottomTabNavigator}
-            options={{ headerShown: false }}
+            options={{ headerShown: false, gestureEnabled: false }}
```

Comment:
@jamccomb92 this fixes the problem where the user could swipe left or right and it would take them to the login screen